OpenTabletDriver.UX/Dialogs/OfflineAboutDialog.cs

```diff
@@ -1,23 +0,0 @@
-using Eto.Forms;
-
-namespace OpenTabletDriver.UX.Dialogs
-{
-    public class OfflineAboutDialog : AboutDialog
-    {
-        public OfflineAboutDialog()
-        {
-            Title = "OpenTabletDriver";
-            ProgramName = "OpenTabletDriver";
-            ProgramDescription = "Open source, cross-platform tablet configurator";
-            WebsiteLabel = "OpenTabletDriver";
-            Website = new Uri(@"https://opentabletdriver.net");
-            Version = $"v{Metadata.FullVersion}";
-            Developers = new[] { "InfinityGhost" };
-            Designers = new[] { "InfinityGhost" };
-            Documenters = new[] { "InfinityGhost" };
```

Comment:
Reimplement these in Build.props for an about tab or about dialog? The actual content can be updated later